src/test/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceServiceTest.java

```diff
@@ -108,8 +113,208 @@ public void getObjects() throws Exception {
+        assertTrue(response.size() == 0);
+
+    }
+
+    @Test
+    public void findByQueryParametersLastModified() throws Exception
+    {
+        final UUID accountUuid = UuidUtil.getNewUuid();
+        final String accountUrn = UuidUtil.getAccountUrnFromUuid(accountUuid);
+        Map<String, Object> queryParams = new HashMap<>();
+
+        Long firstDate = new Date().getTime();
+        Thread.sleep(DELAY_BETWEEN_LAST_MODIFIED_DATES);
+
+        ObjectEntity firstObject = ObjectEntity.builder().accountId(accountUuid)
+            .objectUrn("objectUrnLastModTest1").name("last mod test 1").type("anythingIsFine").build();
+        this.objectRepository.save(firstObject);
+
+        Long secondDate = new Date().getTime();
+        Thread.sleep(DELAY_BETWEEN_LAST_MODIFIED_DATES);
+
+        ObjectEntity secondObject = ObjectEntity.builder().accountId(accountUuid)
+            .objectUrn("objectUrnLastModTest2").name("last mod test 2").type("anythingIsFine").build();
+        this.objectRepository.save(secondObject);
+
+        Long thirdDate = new Date().getTime();
+        Thread.sleep(DELAY_BETWEEN_LAST_MODIFIED_DATES);
+
+        ObjectEntity thirdObject = ObjectEntity.builder().accountId(accountUuid)
+            .objectUrn("objectUrnLastModTest3").name("last mod test 3").type("anythingIsFine").build();
+        this.objectRepository.save(thirdObject);
+
+        Long fourthDate = new Date().getTime();
+
+        queryParams.put(ObjectPersistenceService.MODIFIED_AFTER, firstDate);
+        List<ObjectResponse> response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 3);
+
+        queryParams.put(ObjectPersistenceService.MODIFIED_AFTER, secondDate);
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 2);
+
+        queryParams.put(ObjectPersistenceService.MODIFIED_AFTER, thirdDate);
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 1);
+
+        queryParams.put(ObjectPersistenceService.MODIFIED_AFTER, fourthDate);
+        response = objectPersistenceService
+            .findByQueryParameters(accountUrn, queryParams);
+        assertTrue(response.size() == 0);
+
+    }
+
+    // used by findByQueryParametersStringParameters()
+    private void findByQueryParametersUtil() throws Exception
+    {
+
+        final UUID accountUuid = UuidUtil.getNewUuid();
+
+        ObjectEntity entityNameOneTypeOne = ObjectEntity.builder().accountId(accountUuid)
+            .objectUrn("objectUrnQueryParams01").name("name one").type("type one").build();
+
+        ObjectEntity entityNameTwoTypeOne = ObjectEntity.builder().accountId(accountUuid)
+            .objectUrn("objectUrnQueryParams02").name("name two").type("type one").build();
```

Comment:
If you are using this in more than one location a `private static final String` style constant would be best.  Reusing a constant will reduce the possibilities for false success/failure.